generator/ServiceClientGeneratorLib/GeneratorDriver.cs

```diff
@@ -153,9 +153,9 @@ public void Execute()
                 Directory.CreateDirectory(GeneratedFilesRoot);
             }
 
-            // .NET Framework 4.5 version
-            ExecuteGenerator(new ServiceClients45(), "Amazon" + Configuration.ClassName + "Client.cs", Bcl45SubFolder);
-            ExecuteGenerator(new ServiceInterface45(), "IAmazon" + Configuration.ClassName + ".cs", Bcl45SubFolder);
+            // .NET Framework version
+            ExecuteGenerator(new ServiceClientsNetFramework(), "Amazon" + Configuration.ClassName + "Client.cs", Bcl45SubFolder);
```

Comment:
I'll rename the `Bcl45SubFolder` variable when going through the `BCL` | `BCL35` constants.